common/src/main/java/com/wynntils/wynn/model/questbook/QuestBookManager.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.model.questbook;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.managers.CoreManager;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wynn.event.QuestBookReloadedEvent;
+import com.wynntils.wynn.event.WorldStateEvent;
+import com.wynntils.wynn.model.WorldStateManager;
+import com.wynntils.wynn.model.container.ContainerContent;
+import com.wynntils.wynn.model.container.ScriptedContainerQuery;
+import com.wynntils.wynn.utils.ContainerUtils;
+import com.wynntils.wynn.utils.InventoryUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class QuestBookManager extends CoreManager {
+    private static final Pattern QUEST_NAME_MATCHER = Pattern.compile("^§.§l([^֎À]*)[֎À]+ (§e\\[Tracked\\])?$");
+    private static final int NEXT_PAGE_SLOT = 8;
+
+    private static List<QuestInfo> quests = List.of();
+    private static List<QuestInfo> newQuests;
+
+    public static void init() {}
+
+    @SubscribeEvent
+    public static void onQuestsReloaded(QuestBookReloadedEvent event) {
+        McUtils.player().sendMessage(new TextComponent("Quest Book scanned.").withStyle(ChatFormatting.GRAY), null);
```

Comment:
Do we really need this? I know Legacy did, but that was since it was slow and clumsy, and not always done at the right moment. Now we rescan everytime we open the quest book.

Reply:
I am not sure, it can indicate that something is wrong.

Reply:
Then maybe send the error message in case of failure to the chat instead of the log?